packages/block-editor/src/components/use-block-drop-zone/index.js

```diff
@@ -521,7 +542,18 @@ export default function useBlockDropZone( {
 			// https://developer.mozilla.org/en-US/docs/Web/API/Event/currentTarget
 			throttled( event, event.currentTarget.ownerDocument );
 		},
-		onDragLeave() {
+		onDragLeave( event ) {
+			const { ownerDocument } = event.currentTarget;
+
+			// If the drag event is leaving the drop zone and entering an insertion point,
+			// do not hide the insertion point as it is conceptually within the dropzone.
+			if (
+				isInsertionPoint( event.relatedTarget, ownerDocument ) ||
+				isInsertionPoint( event.target, ownerDocument )
+			) {
+				return;
```

Comment:
I'm not sure of the answer here, so this is just a question! Before the `return;` statement, should we call `throttled.cancel()` to cancel the next cycle of that call, or does it not matter? I'm guessing we're treating this return statement as though nothing happened so _not_ calling `throttled.cancel()` does seem logical to me 👍

Reply:
> we're treating this return statement as though nothing happened

That's how I"m seeing it 👍